AdvancedCalculator/Infrastucture/Calculators/ExpandedCaclulator.cs

```diff
@@ -64,5 +64,21 @@ public bool PerfectNumber(int number)
                 return false;
             
         }
+        public int AverageOfNumbersArray(int arry)
```

Comment:
Średnia nazwa, jak zwracamy coś to powinno być bardziej np GetAvgNumberFromArray jak juz, no i parametr int arry także słaby, nazwij to value

Reply:
Poprawione.